typings/index.d.ts

```diff
@@ -4309,7 +4309,7 @@ export type WSEventType =
   | 'STAGE_INSTANCE_UPDATE'
   | 'STAGE_INSTANCE_DELETE';
 
-export type Serialized<T> = T extends symbol | bigint | (() => unknown)
+export type Serialized<T> = T extends symbol | bigint | (() => any)
```

Comment:
Same, should be unknown, or the function should be turned into a generic one? CC @kyranet

Reply:
Doesn't matter because we're not using the output.